sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -458,13 +458,15 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
   ): Promise<void> {
     return this.dataSource.transaction(async (transactionalEntityManager) => {
       // create the note for assessment.
-      const user = {
+      const auditUser = {
```

Comment:
not part of the PR. but starts with caps    ` // Create the note for assessment.`